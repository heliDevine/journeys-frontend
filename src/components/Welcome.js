const Welcome = () => {
  return (
    <div>
      <h1>Welcome to the App</h1>
      <p>This is the welcome text component.</p>
    </div>
  );
};

export default Welcome;
