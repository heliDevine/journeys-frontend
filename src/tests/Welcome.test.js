import React from 'react';
import { render } from '@testing-library/react';
import Welcome from '.././components/Welcome';

describe('Welcome', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Welcome />);

    expect(asFragment(<Welcome />)).toMatchSnapshot();
  });
});
