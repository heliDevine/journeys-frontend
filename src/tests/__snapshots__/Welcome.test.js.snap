// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome renders correctly 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Welcome to the App
    </h1>
    <p>
      This is the welcome text component.
    </p>
  </div>
</DocumentFragment>
`;
